### ISSUE TITLE
NTR positive regulation of androgen receptor signaling pathway

### ISSUE THREAD

**Issue by Antonialock** [geneontology/go-ontology#16634]
NTR positive regulation of androgen receptor signaling pathway
child of GO:0060765
standard def
PMID:25359680

**Comment by RLovering** [comment]
Hi

FYI UCL would probably not annotate this paper because the cell line used is a cancer cell line, therefore the biological processes and signaling pathways might not be 'normal'
Ideally look for information about the role of this gene in normal tissue.
Were you planning to annotate the genes identified in the siRNA screen too?

Ruth

**Comment by Antonialock** [comment]
Thanks Ruth, I'm really grateful for your expert input. 

Overall I have been cautious when reading papers using cancer cell lines (from your advice in workshops :-) ). 

Currently I have 4 annotations from this paper

primary name | uniprot ID | GOID | GO term | evidence |  with | extension  | ref
-- | -- | -- | -- | -- | -- | -- | --
HSPBAP1 | Q96EW2 | GO:0000790 | nuclear chromatin | IDA |   |   | PMID:25359680
HSPBAP1 | Q96EW2 | GO:0005515 | protein binding | IPI | P10275 |   | PMID:25359680
AR | P10275 | GO:0005515 | protein binding | IPI | Q96EW2 |   | PMID:25359680
HSPBAP1 | Q96EW2 | GO:NEW | positive regulation of androgen receptor signaling pathway | IPI | P10275 |   | PMID:25359680

These annotations are from the following evidence:

Protein binding: evidence from co-IPs  and 2. "antibody-based PLAs" ("which is a method for analysis of endogenous single molecule protein interactions with fluorescence detection") - I'd assume this means that they can interact directly and hence made the protein binding annotations.

nucleus: A distinct PLA signal positivity was identified mainly in the nuclei of VCaP cells

positive regulation of androgen receptor signaling pathway:
 - The interactions of HSPBAP1 and AR, specifically under hormone-deprived conditions, lead us to speculate on a role for HSPBAP1 as a transcriptional co-activator maintaining AR signaling in the absence of androgen. SPBAP1 belongs to the family of histone demethylases containing a conserved jumonji C domain. Previously, five other jumonji domains containing histone demethylases have been demonstrated to associate  with AR. 
 - To analyze the effect of these knockdowns on AR target genes, we performed real-time qPCR on the same samples for the androgen-regulated genes PSA, TMPRSS2, ORM1, MYBPC1, NDRG1, NKX3, FASN and NOV. Our results show up to 50% reduction of PSA and TMPRSS2 mRNA in androgen-deprived conditions in both cell lines following HSPBAP1 knockdown when compared to the scrambled control (Fig.4d). The induction seen on these target genes by R1881 treatment  (4–48  hr)  is  similarly  reduced  up  to  50%  upon HSPBAP1  knockdown.
 - Taken together, our results suggest that the increased interaction between HSPBAP1 and AR in nuclei is likely to occur on DNA and has a regulatory impact upon AR-mediated transcriptional activity especially under androgen deprivation

 - nuclear chromatin: The HSPBAP1 influence on the regulation of AR target led us to investigate if HSPBAP1 is able to bind chromatin. Therefore,  we  performed  ChIP  experiments  of  AR  and HSPBAP1 at the enhancer region of PSA and TMPRSS2 in VCaP and LNCaP cells (Fig. 4e). As expected, the level of chromatin binding of AR is markedly increased in R1881-treated samples. HSPBAP1 occupancy is not increased with androgen treatment but found consistently on chromatin

What do you think?

**Comment by Antonialock** [comment]
there seemed to be quite a bit of biology there, but of course it could just be 'disease biology'...

**Comment by Antonialock** [comment]
also the protein has been shown to be expressed in the thymus (PMID:11978969) and the thymus is a target of androgenic hormones (PMC3077079)  -very circumstantial I know! but at least it doesn't imply that it is completely bonkers? 

Very happy to remove though if you think the evidence is not credible

**Comment by RLovering** [comment]
Hi Antonia
I think the majority of your annotations are fine. 
I would annotate to nuclear chromatin, and also possibly cytoplasm - depending on the figures.

What can be a concern with annotation of cancer cells are statements like: The transcriptional regulation by AR is mediated by association with distinct co‐regulators (co‐activators/co‐repressors) that are often aberrantly expressed in cancer patients.
And also from the discussion: Such co‐regulator complexes would aid in sustaining cell viability by maintaining AR signaling in androgen‐depleted conditions. 
Thus can you be sure that the co-regulator is normally expressed in a tissue that is also expressing the androgen receptor, or if it is expressed in normal cells are there other factors that will influence whether this protein will act as a co-regulator of AR signaling? 

Also they have stated: HSPBAP1 belongs to the family of histone demethylases containing a conserved jumonji C domain. Previously, five other jumonji domains containing histone demethylases have been demonstrated to associate with AR. So then you need to consider if this protein having an effect because it is being over expressed in these cells to such a high level that although it has low affinity with AR it is out competing those proteins with higher affinity - but are present at lower levels? it would have been intersting to see co-Ippts with some of the other proteins.

Figure 4 is a bit concerning. Fig 4d data is presented as a heat map, why not as a graph or histogram? and we are advised that: Our results show up to 50% reduction of PSA and TMPRSS2 mRNA in androgen‐deprived conditions in both cell lines following HSPBAP1 knockdown when compared to the scrambled control (Fig. 4d). The data looks resonable, but for the other data it doesn't look quite as clean, and there is not statement that these changes are 'significant' and often scientist are now being asked to provide 6 replicates of data rather than 3, this only provides 2 replicates. Figure 4e shows no increase in HSPBAP1 at enhancers following androgen stimulation, in contrast to the change of AR at these sites.

Another thought is the paper title:Androgen receptor‐interacting protein HSPBAP1 facilitates growth of prostate cancer cells in androgen‐deficient conditions. Can we describe this as AR signaling if no androgen is added? In addition, what is being measured is gene expression, could this just be upregulation of gene expression in general?

The PPI data looks fine, although note that both bind HSP27 so there is a chance the interaction is not direct. I would use the more specfic term:  GO:0050681 androgen receptor binding

PMID:10751411 describes the rat ortholog PASS1, which is predominantly expressed in testis, this adds more weight to the AR annotation. I would probably annotate this paper to create  GO:0051008 
Hsp27 protein binding annotation which could be ISS's to the human gene, and potentially add 'testis' or similar as an AE if possible, and the human HSP27 ID to the AE field for the human ISS'd annotation.

The problem is that with 1000's of papers to annotate I don't challenge myself by trying to annotate papers like this. It is easier to just say let's avoid these than to try to work out how to capture this data. What is needed is recognition that this data needs to be captured and to have a database to use GO terms to capture disease processes.

In summary, I don't think I would make the AR signaling annotation. But maybe Im being over cautious

Ruth

**Comment by vanaukenk** [comment]
@Antonialock 
I am curious about what led you to this paper.  As we try to develop more curation guidelines it is helpful to know why curators might choose certain papers to curate over others.  Thx.

**Comment by Antonialock** [comment]
@vanaukenk 

We have been working on a paper describing unknown proteins. We noticed that historically, there was a period of rapid characterization of proteins in both in fission and budding yeast, but new characterizations plateaued around a decade ago at just over 80%. Since yeasts serve as a model for eukaryotic biology, we were interested to see how this number compared to human. We therefore extended our yeast slim and found that human had around the same number of unknowns as yeast as a proportion of all proteins. 

For cerevisiae and pombe all unknowns are "ND", but this was not the case for human (I think only around 200 odd proteins of the human unknowns had ND annotation). I therefore performed a literature search for the human unknowns without ND annotation to see if they had been published or not, and if they had whether I could get a BP annotation that maps to the slim out of the paper. In this case specific case pubmed retrieves 9 papers when searching for HSPBAP1 https://www.ncbi.nlm.nih.gov/pubmed/?term=HSPBAP1
Most of them are very obviously not relevant for cell biology, but I thought the last section of PMID:25359680 had some decent  protein characterization in it (I still need to process Ruths comment above).

**Comment by vanaukenk** [comment]
Thanks so much @Antonialock 
I also just commented on #16638 since I thought perhaps this was part of the project to annotate the human unknowns.
These might be helpful papers to discuss and/or use as examples for how to decide if and what information can be captured.

**Comment by ValWood** [comment]
Antonia was checking the list we defined as "unknown" in a recent paper  to see if any were annotatable.

I guess  HSPBAP1
https://www.uniprot.org/uniprot/Q96EW2
was in the list.

There are only 9 papers in PubMed referring to  HSPBAP1 and looking at those, this paper seems to be the most informative. If this isn't enough information to make an annotation we can include this in the "unknown process" bin.

These weren't really the unknowns I was trying to capture. We really only needed to make sure that we had made  or suggested "biological process" annotations for the  ones which  had product lines/descriptions indicating they were definitely well-known, because reviewers would be likely to question if we included them in the "unknowns bin".

**Comment by ValWood** [comment]
Either the function section or the protein label.
So for example this was in our original list
https://www.uniprot.org/uniprot/Q5UCC4
because it had the protein label  "ER membrane protein complex subunit 10"
although it now has annotation dated September, I don't think all the annotations likely reflect ECM10's real cellular role......

https://www.ebi.ac.uk/QuickGO/term/GO:0072546
It is involved in lipid transfer from the mitochondria to the ER

**Comment by ValWood** [comment]
These are  better example:
Still unnannotated:
https://www.uniprot.org/uniprot/Q9NRP2
https://www.uniprot.org/uniprot/Q9BQ75
https://www.uniprot.org/uniprot/Q5SRI9 (an N-glycan trimming alpha-mannosidase, has MF but no BP)

**Comment by vanaukenk** [comment]
Thanks @ValWood 
These would be good examples to look at and discuss on an annotation call.

**Comment by ukemi** [comment]
Do we still need this term added to the ontology? If not, I will close this ticket.
